src/libs/actions/Policy/Tag.ts

```diff
@@ -846,6 +847,90 @@ function setPolicyTagGLCode(policyID: string, tagName: string, tagListIndex: num
     API.write(WRITE_COMMANDS.UPDATE_POLICY_TAG_GL_CODE, parameters, onyxData);
 }
 
+function setPolicyTagApprover(policyID: string, tag: string, approver: string) {
+    const policy = PolicyUtils.getPolicy(policyID);
+    const prevApprovalRules = policy?.rules?.approvalRules ?? [];
+    const approverRuleToUpdate = PolicyUtils.getTagApproverRule(policyID, tag);
+    const filteredApprovalRules = approverRuleToUpdate ? prevApprovalRules.filter((rule) => rule.id !== approverRuleToUpdate.id) : prevApprovalRules;
+    const toBeUnselected = approverRuleToUpdate?.approver === approver;
+    const tagRuleUpdateKey = PolicyUtils.getPolicyFieldUpdateKey(tag, CONST.POLICY.FIELDS.TAG);
```

Comment:
I thought about this, but if we're trying to use it for multiple use cases (categories, tags etc) then this might enforce this to be uniform in all of them and avoid a typo when copypasting it inside of actions, offline wrappers and so on. If it wasn't for the multiple file, where those keys are being used, I'd avoid it. I don't have a strong opinion on that, so up to you 🤷

Reply:
I'm good with it as long as it has a more descriptive name for the function and some explanation of how to use it.